CSA.md

```diff
@@ -1,215 +1,221 @@
+    10.2 <span class="header_3">Protection by Customer</span>.  <span class="keyterms_link">Customer</span> will indemnify, defend, and hold harmless <span class="keyterms_link">Provider</span> from and against all <span class="keyterms_link">Customer Covered Claims</span> made by someone other than <span class="keyterms_link">Provider</span> or its Affiliates, and <span class="keyterms_link">Provider</span> for all out-of-pocket damages, awards, settlements, costs, and expenses, including reasonable attorneys' fees and other legal expenses, that arise from the <span class="keyterms_link">Customer Covered Claim</span>.
 
     10.3 <span class="header_3">Procedure.</span>  The Indemnifying Party's obligations in this section are contingent upon the Protected Party: (a) promptly notifying the Indemnifying Party of each Covered Claim for which it seeks protection; (b) providing reasonable assistance to the Indemnifying Party at the Indemnifying Party's expense; and (c) giving the Indemnifying Party sole control over the defense and settlement of each Covered Claim. A Protected Party may participate in a Covered Claim for which it seeks protection with its own attorneys only at its own expense. An Indemnifying Party may not agree to any settlement of a Covered Claim that contains an admission of fault or otherwise materially and adversely impacts the Protected Party without the prior written consent of the Protected Party.
 
-    10.4 <span class="header_3">Changes to Cloud Service.</span>  If required by settlement or court order, or if deemed reasonably necessary in response to a <span class="coverpage_link">Provider</span> Covered Claim, <span class="coverpage_link">Provider</span> may: (a) obtain the right for <span class="coverpage_link">Customer</span> to continue using the Cloud Service; (b) replace or modify the affected component of the Cloud Service without materially reducing the general functionality of the Cloud Service; or (c) if neither (a) nor (b) are reasonable, terminate the affected Order Form and issue a pro-rated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>.
+    10.4 <span class="header_3">Changes to Product.</span>  If required by settlement or court order, or if deemed reasonably necessary in response to a <span class="keyterms_link">Provider Covered Claim</span>, <span class="keyterms_link">Provider</span> may: (a) obtain the right for <span class="keyterms_link">Customer</span> to continue using the Product; (b) replace or modify the affected component of the Product without materially reducing the general functionality of the Cloud Service; or (c) if neither (a) nor (b) are reasonable, terminate the affected Order Form and issue a pro-rated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>.
```

Comment:
Change "Cloud Service" in (b) to "Product"

```suggestion
    10.4 <span class="header_3">Changes to Product.</span>  If required by settlement or court order, or if deemed reasonably necessary in response to a <span class="keyterms_link">Provider Covered Claim</span>, <span class="keyterms_link">Provider</span> may: (a) obtain the right for <span class="keyterms_link">Customer</span> to continue using the Product; (b) replace or modify the affected component of the Product without materially reducing the general functionality of the Product; or (c) if neither (a) nor (b) are reasonable, terminate the affected Order Form and issue a pro-rated refund of prepaid fees for the remainder of the <span class="orderform_link">Subscription Period</span>.
```

Reply:
I think this one was Cloud Service in the word doc,  but I'll change it.